### ISSUE TITLE
Metadata editing at the nominate level

### ISSUE THREAD

**Issue by AramZS** [PressForward/pressforward#940]

**Comment by jccatalano** [comment]
It appears that this may be impacted by method of nomination. I can edit nominations that came via the bookmarklet but not all content.

**Comment by jccatalano** [comment]
Should users be able to modify the Submitted By field? Since this is based off of user numbers instead of names, I am not sure that this will be a meaningful functionality. Because this is based upon the user number, it is possible that people may confuse this with the number of nominations (not the number(s) of the nominators). If a user can delete or add the numbers of nominators, what impact will that have on the valid of the post metadata for nominations?

**Comment by lcrossley** [comment]
For posts nominated via bookmarklet, when I delete all of the original tags in Nominated and add a new tag, then send the item to draft, the new tag gets added in the draft, but the original tags are still there, too. When I check the metadata in Nominated or try to edit it again, then only the new tag is there.

**Comment by AramZS** [comment]
@jccatalano the changes aren't reflected in the post meta data even after refresh for those two fields?

**Comment by jccatalano** [comment]
Correct. The changes do not appear anywhere even if the nomination is published.

**Comment by jccatalano** [comment]
Should Thumbnail ID, Date Posted on Source, Nomination Datetime, & Date added to PressForward be options for editing?

**Comment by jccatalano** [comment]
There is also a discrepancy between what fields can be edited based upon the nomination method. If editing a post that originated from all content only Date Posted on Source, Author(s), Date added to PressForward, Nomination Datetime, & Thumbnail ID are options to edit. 

Meanwhile, posts via the bookmarklet have Date Posted on Source, Author(s), Date added to PressForward, Nomination Datetime, Thumbnail ID, Origin Item URL, Source Title, & Tags.

**Comment by jccatalano** [comment]
The save button does not work for posts that originate from all content.

**Comment by AramZS** [comment]
Moving this to 5.2
